src/css-optimize.php

```diff
@@ -262,7 +262,8 @@ public function strip_optimized_block_styles( $block_content, $block ) {
 			}
 
 			// Only do this to our blocks.
-			if ( ! empty( $block ) && is_array( $block ) && stripos( $block['blockName'], 'stackable/' ) === 0 ) {
+			$blockName = isset($block['blockName']) ? $block['blockName'] : ''
```

Comment:
@vmc08 a few standards checks here since we don't have a php linter:
- Mind the spacing between parenthesis `isset( $block['blockName'] )`
- Use snake case in php `$block_name` instead of `$blockName`
- Semi-colon at line end `;`